maths/sum_of_arithmetic_series.py

```diff
@@ -0,0 +1,12 @@
+# DarkCoder
+def sumOfSeries(a, d, n):
+    sum = ((n / 2) * (2 * a * (n - 1) * d))  # formula for sum of series
```

Comment:
I believe the formula is `((n/2)*(2*a + (n-1)*d))`. Please also add documentation as to what the parameters are and the return value of the function. Please also consider adding doctests to the function.

Reply:
Also, please use the descriptive variable names.